wordpress-mobile/WordPress-Android
# Issue/8646 quick start completed view

Fixes #8646

This PR adds a "Quick Start Completed" view that appears when all tasks in a list are completed, and the list is collapsed.

Test 1:
- Open Quick Start task list when all the tasks are completed.
- Notice Completed View.
- Toggle the completed tasks list and see that Completed View fades in and out.

Test 2:
- While completed task list is collapsed, skip all the tasks in the list.
- After the last task is skipped notice that the Completed View appears.

Test 3:
- Confirma that AEV illustration are different for GROW and CUSTOMIZE lists.

Visuals (clicking on gif opens HD version):

[![Image from Gyazo](https://i.gyazo.com/1af27c5d7eaa321a4e4199be7f417f20.gif)](https://gyazo.com/1af27c5d7eaa321a4e4199be7f417f20)

[![Image from Gyazo](https://i.gyazo.com/beafecd0526211b768435e89e5f6196f.gif)](https://gyazo.com/beafecd0526211b768435e89e5f6196f)
